src/analyses/libraryFunctions.ml

```diff
@@ -29,6 +37,34 @@ let posix_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("__builtin_bzero", special [__ "dest" [w]; __ "count" []] @@ fun dest count -> Bzero { dest; count; });
     ("explicit_bzero", special [__ "dest" [w]; __ "count" []] @@ fun dest count -> Bzero { dest; count; });
     ("__explicit_bzero_chk", special [__ "dest" [w]; __ "count" []; drop "os" []] @@ fun dest count -> Bzero { dest; count; });
+    ("nl_langinfo", unknown [drop "item" []]);
+    ("nl_langinfo_l", unknown [drop "item" []; drop "locale" [r]]);
+    ("getc_unlocked", unknown [drop "stream" [w]]);
+    ("getchar_unlocked", unknown []);
+    ("putc_unlocked", unknown [drop "c" []; drop "stream" [w]]);
+    ("putchar_unlocked", unknown [drop "c" []]);
+    ("lseek", unknown [drop "fd" [w]; drop "offset" []; drop "whence" []]);
+    ("fseeko", unknown [drop "stream" [w]; drop "offset" []; drop "whence" []]);
+    ("iconv_open", unknown [drop "tocode" [r]; drop "fromcode" [r]]);
+    ("iconv", unknown [drop "cd" [r]; drop "inbuf" [r]; drop "inbytesleft" []; drop "outbuf" [w]; drop "outbytesleft" []]);
```

Comment:
According to the man page, it decrements the `bytesleft` arguments, which is why they're `size_t*`. So they should be `r; w` I think.